templates/careers/company-culture/remote-work.html

```diff
@@ -18,100 +18,117 @@
-        </p>
-        <p>
-          <a href="/careers/career-explorer">What kind of excellent are you&nbsp;&rsaquo;</a>
-        </p>
+        <div class="p-strip is-shallow u-no-padding--top">
+          <h2>We are remote but connected</h2>
+        </div>
+        <div class="p-section--shallow">
+          <p>Canonical has been a fully distributed organization since its origin in 2004.</p>
+          <p>
+            Our remote-first policy enables us to collaborate with the best in the world in our domain. Though most team members work remotely, we meet each other at least twice per year during in-person sprints to nurture cross-team relationships. These sprints are funded by Canonical, and take place at locations around the world.
+          </p>
+        </div>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive u-no-margin--bottom" href="/careers">
+          Careers with Canonical
+        </a>
       </div>
     </div>
   </section>
 
-  <section class="p-section">
+  <section class="p-strip u-no-padding--top">
     <div class="u-fixed-width">
       <hr class="p-rule" />
     </div>
-    <div class="row">
-      <div class="col-6 col-medium-3">
-        <h2>We travel</h2>
+    <div class="row--25-75">
+      <div class="col">
+        <h2>
+          Our work ethic
+        </h2>
       </div>
-      <div class="col-6 col-medium-3">
-        <p>
-          We optimize our team structure for time zone overlap. Collaboration and inspiration takes time to fully develop but leadership at Canonical means building cross-team relationships. We nurture those relationships through regular global summits which bring diverse teams or their leaders together.
-        </p>
-        <p>
-          We host these events around the world to discover great places to explore and appreciate in the process. Team events have taken place in Vancouver, Madrid, Riga, Cape Town, Prague, London, and many more.
-        </p>
+      <div class="col">
+        <div class="row u-no-padding--top u-no-padding--bottom">
+          <div class="col-3 col-medium-1">
+            <h3 class="p-heading--5">
+              We build trust
+            </h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Zero commute means more time for the things you enjoy. At the same time, we have very high expectations of commitment and independent execution.
+            </p>
+            <p>We invest in people who are self-driven, motivated, and highly organized.</p>
+            <p class="u-sv2">
+              We take our responsibility for the most critical elements of open infrastructure very seriously, and we are committed to quality in every aspect of our product and services.
+            </p>
+            <hr class="p-rule--muted" />
```

Comment:
You can use CTA block for sections like these.

This way you will have the muted border, and correct spacing out of the box.

https://vanillaframework.io/docs/patterns/cta-block